src/govuk/components/header/template.test.js

```diff
@@ -139,6 +139,20 @@ describe('header', () => {
       expect($navigationLink.html()).toContain('<em>Nav item</em>')
     })
 
+    it('renders navigation item with text without a link', () => {
```

Comment:
It might also be a good idea to test that when the navigation item is defined with `text` that entities are escaped like we expect them to be.

We do something similar here:

https://github.com/alphagov/govuk-frontend/blob/4a4d9342999cd559317724a6054b1b89da0f8d1e/src/govuk/components/label/template.test.js#L55-L62

Reply:
Good point about there not being a test to check that navigation item text is escaped although it's slightly outside of what this PR was meant to. But I've added a commit to improve the existing test coverage.